frontend/src/routes/_layout.tsx

```diff
@@ -0,0 +1,185 @@
+      }
+
+      queryClient.removeQueries({
+        queryKey: userKeys.authedUser
+      });
+      deleteCookie("csrftoken");
+      navigate({ to: "/login" });
+      return null;
+    }
+  });
+
+  if (!user) {
+    return null;
+  }
+  return (
+    <>
+      <header className="flex px-6 border-b border-opacity-65 border-border py-2 items-center bg-toogle t justify-between gap-4">
+        <Logo className="w-10 flex-none h-10 mr-8" />
+        <div className="flex  w-full items-center">
+          <Menubar className="border-none w-fit text-black bg-primary">
+            <MenubarMenu>
+              <MenubarTrigger className="flex justify-center text-sm items-center gap-1">
+                Create
+                <ChevronsUpDown className="w-4" />
+              </MenubarTrigger>
+              <MenubarContent className="border min-w-6 border-border">
+                <MenubarContentItem icon={Folder} text="Project" />
+                <MenubarContentItem icon={Globe} text="Web Service" />
+                <MenubarContentItem icon={Hammer} text="Worker" />
+                <MenubarContentItem icon={AlarmCheck} text="CRON" />
+              </MenubarContent>
+            </MenubarMenu>
+          </Menubar>
+          <div className="flex w-full justify-center items-center">
+            <Search className="relative left-10" />
+            <Input
+              className="px-14 my-1 text-sm focus-visible:right-0"
+              placeholder="Search for Service, Worker, CRON, etc..."
+            />
+          </div>
+          <HelpCircle className="w-16 stroke-[1.5px] opacity-70" />
+        </div>
+
+        <Menubar className="border-none w-fit">
+          <MenubarMenu>
+            <MenubarTrigger className="flex justify-center items-center gap-2">
+              <CircleUser className="w-5 opacity-70" />
+              <p>{user.username}</p>
+              <ChevronDown className="w-4 my-auto" />
+            </MenubarTrigger>
+            <MenubarContent className="border min-w-0 mx-9  border-border">
+              <MenubarContentItem icon={Settings} text="Settings" />
+              <button
+                onClick={() => mutate()}
+                type="submit"
+                className="w-full"
+                disabled={isPending}
+              >
+                {isPending ? (
+                  "Logging out..."
+                ) : (
+                  <MenubarContentItem icon={LogOut} text="Logout" />
+                )}
+              </button>
+            </MenubarContent>
+          </MenubarMenu>
+        </Menubar>
+      </header>
+    </>
+  );
+}
+
+const linksIconWidth = 15;
+const links = [
+  { name: "Feedback", url: "", icon: <Send width={linksIconWidth} /> },
+  { name: "Docs", url: "", icon: <BookOpen width={linksIconWidth} /> },
+  {
+    name: "Contribute",
+    url: "",
+    icon: <HeartHandshake width={linksIconWidth} />
+  },
+  { name: "Twitter", url: "", icon: <Twitter width={linksIconWidth} /> }
+];
```

Comment:
The links are :


- Docs : https://github.com/zane-ops/zane-ops/blob/main/docs.md (temporary)
- Feedback : https://github.com/zane-ops/zane-ops/discusssions (temporary)
- Contribute : https://github.com/zane-ops/zane-ops/blob/main/CONTRIBUTING.md
- Twitter : https://twitter.com/zaneopsdev